presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -274,6 +274,7 @@ public final class SystemSessionProperties
     public static final String REWRITE_CROSS_JOIN_OR_TO_INNER_JOIN = "rewrite_cross_join_or_to_inner_join";
     public static final String REWRITE_CROSS_JOIN_ARRAY_CONTAINS_TO_INNER_JOIN = "rewrite_cross_join_array_contains_to_inner_join";
     public static final String REWRITE_LEFT_JOIN_NULL_FILTER_TO_SEMI_JOIN = "rewrite_left_join_null_filter_to_semi_join";
+    public static final String ADD_PARTIAL_NODE_FOR_ROW_NUMBER_NODE_WITH_LIMIT = "add_partial_node_for_row_number_node_with_limit";
```

Comment:
It feels like this is always useful. Is the session property only for controlling rollout?
Also rename to something simpler like:
`use_partial_row_number_with_limit`

Using the word node 2 times in the name makes it sound somewhat redundant